tee-worker/identity/client-sdk/packages/client-sdk/src/lib/requests/get-omni-graph.ts

```diff
@@ -0,0 +1,34 @@
+import type { LitentryIdentity, MemberAccount } from '@litentry/parachain-api';
+
+import type { ApiPromise } from '@polkadot/api';
+import { encodeAddress } from '@polkadot/util-crypto';
+
+/**
+ * Retrieve the omni account graph for a given identity.
+ * @param api - Polkadot API instance
+ * @param who - Litentry identity to query
+ * @returns Array of member accounts or null if not found
+ * @throws Error if API calls fail
+ */
+export async function getOmniGraph(
+  api: ApiPromise,
+  { who }: { who: LitentryIdentity }
+): Promise<MemberAccount[] | null> {
+  try {
+    const omniAccount = await api.rpc.state.call('OmniAccountApi_omni_account', who.toHex());
+
+    const omniAccountAddress = encodeAddress(omniAccount.toHex());
+
+    const accountStore = await api.query.omniAccount.accountStore(omniAccountAddress);
+
+    if (accountStore.isNone) {
+      return null;
+    }
+
+    return accountStore.unwrap() as MemberAccount[];
+  } catch (e) {
+    // swallow - empty omni graph
```

Comment:
Is `accountStore.unwrap()` the only call that can throw an error ?
If not it might ignore it (for example query error) and return null so user will think that account doesn't exists